src/modules/home/Home.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
-import Navbar from './navbar';
-import Footer from './footer';
+import Navbar from '../../shared/components/navbar';
+import Footer from '../../shared/components/footer';
 import { Ecosystem, Features, Search, RecommendedTemplates } from './sections';
 
```

Comment:
You can import like { Navbar, Footer } from shared/components. Just add exports in shared/components/index.ts file  - remember about alphabetic order. Add enter after Footer import - different sections should be separated wioth enter